GoB_2-72.py

```diff
@@ -524,8 +540,8 @@ def execute(self, context):
             varTime = ficTime
         else:
             if importToggle:
-                self.report({'INFO'},"Nothing to update")
-                print("Nothing to update")
+                if verbose == True:
```

Comment:
[PEP8 recommends](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) just testing `if blah:` for truthy values, it's a little neater. Not a big deal, just thought it might be worth mentioning.

Reply:
ah yes that is true, im a bit rusty with my scripting practice.